src/scripts/main.js

```diff
@@ -2,10 +2,28 @@
 
 function waitFor(element, eventName) {
   // write your code here
+  return new Promise((resolve, reject) => {
+    if (!element) {
+      const error = 'Something went wrong';
+
+      reject(error);
+    } else {
+      element.addEventListener(`${eventName}`, (ev) => {
```

Comment:
Using template literals for `eventName` is unnecessary here. You can directly pass `eventName` without interpolation.